tests/plugins/easyimage/manual/datauriimages.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.13.1, bug, 3524
+@bender-ui: collapsed
+@bender-ckeditor-plugins: sourcearea, wysiwygarea, floatingspace, toolbar, easyimage, undo
+@bender-include: ../_helpers/tools.js
+
+1. Open editor's console if possible
+2. Copy-paste or drag and drop image to the editor. Please notice that both image should be processed differently.
```

Comment:
Drag and drop doesn't work in IE11:

![ei-mm ie11](https://user-images.githubusercontent.com/1061942/66203657-3126d200-e6a9-11e9-98de-0da17d4bfecb.gif)

And Edge... well:

![ei-mm edge](https://user-images.githubusercontent.com/1061942/66203675-3a17a380-e6a9-11e9-9393-122b73aab254.gif)

☝️ Most probably visible issues here are not related to the fix, but still worth checking.

Reply:
I extracted the case to #3546. It's a clipboard issue, where images with the dataURI are wrongly processed on IE11.

Reply:
Today I noticed there is also another bug reported here related to Edge browser. I analyzed it and report in #3568.

Reply:
Ok, thanks for extracting those issues 👍